ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -7,6 +7,7 @@
    [cmr.common.services.errors :as errors]
    [cmr.common.time-keeper :as time-keeper]
    [cmr.common.validations.json-schema :as js]
+   [cmr.ingest.api.core :as api-core]
```

Comment:
Services namespaces should not require API namespaces (only the other direction).

I'd move the `set-user-id-for-bulk-update` function into the `ingest-app/src/cmr/ingest/api/bulk.clj` namespace since it is not a core function - only related to the bulk update API. Instead of passing headers into all the service functions you should pass in the user-id.

Reply:
Thank you. I just focused on getting it to work and forgot about the big picture.

Reply:
Done.